compiler-rt/lib/interception/interception_type_test.cpp

```diff
@@ -12,17 +12,26 @@
 //===----------------------------------------------------------------------===//
 
 #include "interception.h"
+#include "sanitizer_common/sanitizer_type_traits.h"
 
-#if SANITIZER_LINUX || SANITIZER_APPLE
```

Comment:
This change is unrelated, but IMO we should check these static asserts everywhere possible.